common/src/com/thoughtworks/go/config/materials/git/GitMaterial.java

```diff
@@ -209,18 +209,24 @@ private GitCommand getGit(File workingdir, int preferredCloneDepth, SubprocessEx
         }
     }
 
+    private boolean isRepository(GitCommand gitCommand, GitRepository gitRepository) {
```

Comment:
Do not migrate non bare repositories as this will be dead code soon and be unpredictable for large repos.